liveclasses/models.py

```
@@ -0,0 +1,64 @@
+    ]
+
+    course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name="live_classes")
+    teacher = models.ForeignKey(User, on_delete=models.CASCADE, related_name="live_classes")
+    topic = models.CharField(max_length=255)
+    meeting_id = models.CharField(max_length=50, unique=True, blank=True)  # Unique Jitsi meeting ID
+    scheduled_at = models.DateTimeField()
+    duration = models.IntegerField(default=60)  # Duration in minutes
+    status = models.CharField(max_length=10, choices=STATUS_CHOICES, default="Upcoming")
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def save(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

Add return type annotation: `None`

(ANN201)